charts/sentry-apps/values.yaml

```diff
@@ -53,6 +53,12 @@ kafka:
   # kafka.transactionStateLogMinIsr -- overridden min.insync.replicas config for the transaction topic
   transactionStateLogMinIsr: 1
 
+  # nginx -- NGINX configuration (required when `ingress.enabled=false`)
```

Comment:
```suggestion
# nginx -- NGINX configuration (required when `ingress.enabled=false`)
```

Reply:
I should deactivate that auto-save formatting hook for good. Thanks